src/license.ts

```diff
@@ -6,12 +6,21 @@ import chalk from "chalk";
 import Debug from "debug";
 import path from "path";
 import parse from "spdx-expression-parse";
-import satisfies from "spdx-satisfies";
+import spdxSatisfiesOriginal from "spdx-satisfies";
 
 import { LicenseStatus, Literals } from "./enumerations";
 import { Configuration, License, NpmPackage, OldLicenseFormat, Package } from "./interfaces";
 import * as util from "./util";
 
+const satisfies = (spdxLicense: string, str: string) => {
```

Comment:
@mvayngrib Rename to `(allowedLicenses: string, license: string)`

Reply:
720b47559dad389bdacbd9490e03cfb4ae3faa1f